hoomd/md/PotentialBondGPU.h

```diff
@@ -64,11 +69,14 @@ class PotentialBondGPU : public PotentialBond<evaluator>
     };
 
 template<class evaluator,
-         hipError_t gpu_cgbf(const kernel::bond_args_t& bond_args,
+         class Bonds,
+         int group_size,
+         hipError_t gpu_cgbf(const kernel::bonds_args_t<group_size>& bond_args,
                              const typename evaluator::param_type* d_params,
                              unsigned int* d_flags)>
-PotentialBondGPU<evaluator, gpu_cgbf>::PotentialBondGPU(std::shared_ptr<SystemDefinition> sysdef)
-    : PotentialBond<evaluator>(sysdef)
+PotentialBondGPU<evaluator, Bonds, group_size, gpu_cgbf>::PotentialBondGPU(
```

Comment:
Would it be possible to derive `group_size` at compile time from the `Bonds` class? That would reduce the number of template arguments and limit the chance for error.